packages/wp-source/src/actions.ts

```diff
@@ -122,6 +122,14 @@ const actions: WpSource["actions"]["source"] = {
 
     libraries.source.api.init({ api, isWpCom });
 
+    // If the URL contains an auth token, then add it to the state.
+    // This is normally the case e.g, when accessing the post preview.
+    const auth = state.frontity?.options?.frontity_source_auth;
+    const authFromEnv = process.env.FRONTITY_SOURCE_AUTH;
```

Comment:
Interesting. Which one should take precedence here, the query, or the env variable? 🤔

Reply:
My thinking was that the value from the query string should take precedence over the value defined from the env variable. 

I think that the opposite would not really make sense. It would mean that if you set the FRONTITY_SOURCE_AUTH in your environment then you could not use the `frontity_source_auth` param anymore to set it.

Reply:
I see, thanks.

I was a bit afraid of the opposite case:

- You use the Frontity PHP plugin, which always sends the `frontity_source_auth` query.
- You use the `FRONTITY_SOURCE_AUTH` token to auth with the [JWT plugin](https://github.com/WP-API/jwt-auth) because you prefer that.
- `state.source.auth` is populated with the query.

But that's incidental, just because it happens that in this case, the token in the query has fewer permissions than then token in the env variable. The opposite case could also be true.

In our CLI we decided to use [the env variable only if the argument is not present](https://docs.frontity.org/frontity-cli/environment-variables) so I guess it makes sense to follow the same here.

All good then, thanks Michal!